pkg/ipamd/ipamd.go

```diff
@@ -386,9 +386,8 @@ func (c *IPAMContext) nodeInit() error {
 	}
 	localPods, err := c.getLocalPodsWithRetry()
```

Comment:
TODO - in 1.7 we might want to rename `localPods` to something like `localRunningSandboxes`